pkg/controller/apicast/apicast_logic_reconciler.go

```diff
@@ -63,6 +65,10 @@ func (r *APIcastLogicReconciler) Reconcile() (reconcile.Result, error) {
 		return reconcile.Result{}, nil
 	}
 
+	if r.APIcastCR.Spec.AdminPortalCredentialsRef == nil && r.APIcastCR.Spec.GatewayConfigurationSecret == nil {
```

Comment:
Here I check that one of the secrets must be set

Reply:
see above